renovate.json

```diff
@@ -1,5 +1,22 @@
 {
   "extends": [
     "config:base"
+  ],
+  "ignorePaths": ["example-problems/**"],
```

Comment:
Memo: do we need to exclude "boms/cloud-oss-bom/pom.xml"?

Reply:
The cloud-oss-bom is probably the most important one for renovate to update

Reply:
Elliotte prefers to maintain it manually; he wants 2 humans involved before modifying the content of the BOM.